### ISSUE TITLE
kerAliasesNotSupported: Please send this XMP packet to ahuggel@gmx.net

### ISSUE THREAD

**Issue by moon6969** [Exiv2/exiv2#1404]
[kerAliasesNotSupported](https://github.com/Exiv2/exiv2/blob/38dc741f2d807d0f93d9fdfcd3602f9dc623a1ff/src/error.cpp#L144)
N_("Aliases are not supported. Please send this XMP packet to ahuggel@gmx.net `%1', `%2', `%3'") },

Is Andreas still interested in these!?

I wonder if it should instead refer users to raise a Github issue? If not, just "Aliases are not supported".

**Comment by clanmills** [comment]
Yes.  "Aliases are not supported" is sufficient.  In the XMP Spec  XMPSpecificationPart2.pdf it says (incomprehensibly):

_Previous versions of this specification referred to aliased properties. Specific XMP implementations may treat a property in one namespace as equivalent to a property in another namespace. However, to foster interchange, applications must always write the standard, “base” form of the property. In this version of the specification, only the base properties are listed._

Alison and I visited and stayed with Andreas and family in 2016.  Andreas has lost all interest in Exiv2.  Time to scrub his name from the code.  I have given him huge praise in the book.  His implementation of the TiffVisitor is very robust, reliable and thorough.

I feel scrubbing the code prologs is something we should do towards the end of releasing Exiv2 v0.28.  Possibly at RC2.  I know that @piponazo would like to reformat all the code using clang based tools.  While we have two shipping branches (0.27-maintenance and master) we have not done this.   However if we ship v0.28 and v0.27.4 in spring, there will be few subsequent changes 0.27-maintenance after v0.27.4.  We haven't had a CVE raised on Exiv2 for more than a year.  There are no open security issues on Exiv2.

```bash
1049 rmills@rmillsmm-local:~/gnu/github/exiv2/0.27-maintenance $ find . -type f -name "*pp" | xargs grep ahuggel
./include/exiv2/crwimage.hpp:           <a href="mailto:ahuggel@gmx.net">ahuggel@gmx.net</a>
./include/exiv2/futils.hpp:           <a href="mailto:ahuggel@gmx.net">ahuggel@gmx.net</a>
...
./src/actions.hpp:           <a href="mailto:ahuggel@gmx.net">ahuggel@gmx.net</a>
./src/convert.cpp:  Author(s): Andreas Huggel (ahu) <ahuggel@gmx.net>
./src/fujimn_int.hpp:           <a href="mailto:ahuggel@gmx.net">ahuggel@gmx.net</a>
./src/error.cpp:          N_("Aliases are not supported. Please send this XMP packet to ahuggel@gmx.net `%1', `%2', `%3'") }, // %1=namespace, %2=property path, %3=value
./src/samsungmn_int.cpp:  Author(s): Andreas Huggel (ahu) <ahuggel@gmx.net>
./src/tiffvisitor_int.cpp:  Author(s): Andreas Huggel (ahu) <ahuggel@gmx.net>
./src/rafimage.cpp:  Author(s): Andreas Huggel (ahu) <ahuggel@gmx.net>
...
```

For that matter, all contributors names should be scrubbed.  Gilles, Brad and others have credited themselves in code prologs.  Author/Contributors are stored in the source code history. 

The only copyright notices in the code (other than "Exiv2 authors") are (and seem legitimate).

```
contrib/organize/organize.cpp: * Copyright (C) 2009 Brad Schick <schickb@gmail.com>
samples/toexv.cpp: * Copyright (C) 2004-2017 Andreas Huggel <ahuggel@gmx.net>
samples/Jzon.cpp:Copyright (c) 2013 Johannes Häggqvist
xmpsdk/include/client-glue/TXMPIterator.incl_cpp:// Copyright 2002-2007 Adobe System
```
